src/skin/firstRun.html

```diff
@@ -1,73 +1,129 @@
-            <li><img src='/icons/slideshow/PB-layouts-05.png' alt=''></li>
-            <li><img src='/icons/slideshow/PB-layouts-06.png' alt=''></li>
-            <li><img src='/icons/slideshow/PB-layouts-07.png' alt=''></li>
-            <li><img src='/icons/slideshow/PB-layouts-08.png' alt=''></li>
-            <li><img src='/icons/slideshow/PB-layouts-09.png' alt=''></li>
-            <li><img src='/icons/slideshow/PB-layouts-10.png' alt=''></li>
-            <li><img src='/icons/slideshow/PB-layouts-11.png' alt=''></li>
-            <li><img src='/icons/slideshow/PB-layouts-12.png' alt=''></li>
-            <li><img src='/icons/slideshow/PB-layouts-12a.png' alt=''></li>
-            <li><img src='/icons/slideshow/PB-layouts-13.png' alt=''></li>
-            <li><img src='/icons/slideshow/PB-layouts-14.png' alt=''></li>
-            <li><img src='/icons/slideshow/PB-layouts-15.png' alt=''></li>
-          </ul>
-        </div>
-        <a href="#" class="jcarousel-control-prev">&lsaquo;</a>
-        <a href="#" class="jcarousel-control-next">&rsaquo;</a>
-        <p class="jcarousel-pagination"></p>
+    <!-- privacy badger settings -->
+    <div id="pb-settings" class="align-center">
+      <div class="text">
+        <img src="images/disable-pb.png" alt="">
+        <p class="i18n_intro_disable_button text"></p>
+        <p class="i18n_intro_report_button"></p>
+        <p class="i18n_intro_privacy_note"></p>
       </div>
     </div>
+    <div class="caret-down align-center">
+      <a href="#pb-donate" class="scroll-it"><img src="images/carrot-down.svg" alt="i18n_next_section"></a>
+    </div>
 
-    <div id="social-links">
-      <h3 class="i18n_intro_social_share"></h3>
-        <a target=_blank href="https://twitter.com/home?status=I%20just%20installed%20Privacy%20Badger%2C%20a%20new%20tool%20from%20@EFF%20to%20stop%20companies%20from%20spying%20on%20your%20browsing%20habits%20https%3A%2F%2Feff.org%2Fprivacybadger&related=eff"><img src="/icons/t.png" title="i18n_share_button_title_twitter"></a>
-        <a href="https://www.facebook.com/share.php?u=https%3A%2F%2Feff.org%2Fprivacybadger" target="_blank"><img src="/icons/f.png" title="i18n_share_button_title_facebook"></a>
-        <a href="https://plus.google.com/share?url=https%3A%2F%2Feff.org%2Fprivacybadger" target=_blank><img src="/icons/g.png" title="i18n_share_button_title_google"></a>
+    <!-- donate ask -->
+    <div id="pb-donate" class="align-center">
+      <div class="text-center">
+        <img src="images/pb-logo-outline.svg" alt="">
+        <h3 class="i18n_intro_donate_heading"></h3>
+        <p class="i18n_intro_donate1 text"></p>
+        <p class="i18n_intro_donate2"></p>
+      </div>
+    </div>
+    <div id="pb-donate-button" class="align-center">
+      <div class="text-center">
+        <a class="i18n_donate_to_eff button large" href="https://supporters.eff.org/donate/support-privacy-badger"></a>
```

Comment:
Open in new tab so they can easily still click on the share buttons without having to reopen the tutorial.